### DISCUSSION TITLE
Editable collection variables in collection settings?

### DISCUSSION THREAD

**Discussion by frixxx** [usebruno/bruno#957] [Ideas]
Hey Guys!

Wouldn't it be great if we could edit collection variables at collection level? (in the settings)

The background to this question is, that we in our team use testdata for specific customers that we use across several api requests (f.e. for debugging of production issues).
At the moment we are populating the collection vars with pre-request scripts on the collection level, but this is pretty messy.

Such a feature would improve this workflow but maybe you have a different take?

Another idea would be to have some kind of testdata tool to switch between different testdata-sets. And have the selected testdata be used in the api requests.

What do you guys think?

Frixxx

**Comment by andrewmnelson** [comment]
A workaround is to create an Environment (upper right corner of the app window) and set global vars there. The vars can be marked 'secret' so they don't get saved to your code repo.

**Reply by acuenca-facephi** [reply]
This workaround forces the variables to be duplicated by each environment.
